hagglehaul.Server/EmailViews/ConfirmationEmail.cshtml

```diff
@@ -1,24 +1,52 @@
 @model hagglehaul.Server.EmailViews.ConfirmationEmail
 
-@{
-    Layout = null;
-}
-
 <!DOCTYPE html>
-
-<html>
+<html lang="en">
 <head>
-    <title>Hagglehaul Trip: Ride Confirmation</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ride Confirmation</title>
     <style>
-        .blue-accent {
-            color: #126391;
+        body {
+            font-family: Arial, sans-serif;
+            background-color: #f4f4f4;
+            padding: 20px;
+        }
+        .container {
+            background-color: #fff;
+            border-radius: 8px;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+            padding: 20px;
+            margin: 20px auto;
+            max-width: 600px;
+        }
+        h1 {
+            color: #333;
+            text-align: center;
+        }
+        p {
+            color: #666;
+            line-height: 1.6;
+        }
+        .footer {
+            margin-top: 20px;
+            text-align: center;
+            color: #999;
         }
     </style>
 </head>
 <body>
-<div>
-    <h1>Confirmation for riders email template</h1>
-    <p class="blue-accent">@Model.ExampleProperty</p>
-</div>
+    <div class="container">
+        <h1>Ride Confirmation</h1>
+        <p>Dear @Model.RiderName,</p>
+        <p>Your ride has been confirmed. Please review the details below:</p>
+        <p><strong>Start Location:</strong> @Model.StartLocation</p>
+        <p><strong>End Location:</strong> @Model.EndLocation</p>
+        <p><strong>Driver:</strong> @Model.DriverName</p>
```

Comment:
add 
* email address for driver
* phone number for driver
* the line that says "Driver:" should not only be the driver name, it should be of the format "Driver Name (4.9 ⭐)" (so add a double for that)
* pickup time
* change "start location" to "pickup location"
* change "end location" to "destination location"

Reply:
@CE201710 this is still missing the rating for the driver 
this is still missing the pickup time.